Formula/osqp.rb

```diff
@@ -0,0 +1,91 @@
+class Osqp < Formula
+  desc "Operator splitting QP solver"
+  homepage "https://osqp.org/"
+  url "https://github.com/oxfordcontrol/osqp/archive/v0.5.0.tar.gz"
+  sha256 "e0932d1f7bc56dbe526bee4a81331c1694d94c570f8ac6a6cb413f38904e0f64"
+
+  depends_on "cmake" => :build
```

Comment:
```suggestion
  depends_on "cmake" => [:build. :test]
```

You need a test dependency on CMake too.

Reply:
I would push this myself if I had permission...

Reply:
Done